src/BlazorUI/Bit.BlazorUI.Extras/Components/RichTextEditor/BitRichTextEditor.scss

```
@@ -1,102 +1,376 @@
+
+    &:focus-visible {
+        outline: spacing(0.25) solid $clr-pri;
+        outline-offset: spacing(-0.25);
+    }
 }
 
 .bit-rte-edt {
-    flex-grow: 1;
+    padding: spacing(1.7) spacing(2);
+    outline: none;
+    // Scroll wide content (notably wide tables) on both axes instead of clipping it, while
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)